generators/entity-client/files.js

```diff
@@ -308,7 +322,7 @@ function writeFiles() {
                 files = reactFiles;
                 destDir = REACT_DIR;
                 templatesDir = CLIENT_REACT_TEMPLATES_DIR;
-            } else {
+            } else if (!this.embedded) {
```

Comment:
You should move this condition to inside else bracket.
Otherwise the method will continue on embedded.

Reply:
Yes you're right! Fixing it right now